### DISCUSSION TITLE
Dimensional issue while running statsforecast.models experiments

### DISCUSSION THREAD

**Discussion by zrqxiangshuijiao** [Nixtla/statsforecast#784] [Q&A]
Hi All,

I am new to Nixtla and currently working on my master thesis.

I have encountered a dimensional issue while running statsforecast.models experiments on a pandas dataframe which has 3 columns.
It shows:  IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed.

Does anyone encounter same or similar issue and happen to know how to fix it?
Thanks in advance!

**Comment by jmoralez** [comment]
Hey @zrqxiangshuijiao, thanks for using statsforecast. Can you provide the code that you're running and also which version of statsforecast you're using?

**Comment by trodlermichal** [comment]
I am having the very same issue. Although running the very same code they have in the tutorial, I always get "IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed". 

This is my code that causes the issue:
fcst = StatsForecast(models=[AutoETS(season_length=12, model='ZZA')], freq='M', n_jobs=-1)
Y_hat_df = fcst.forecast(df=Y_train_df, h=8, fitted=True)
Y_fitted_df = fcst.forecast_fitted_values()

And my dataset was aggregated by aggregate() function and indeed has columns:
"unique_id"
"ds"
"y"
